R/Accelerated_Failure_time_model.qmd

```diff
@@ -0,0 +1,133 @@
+
+Where:
+
+-   T is the survival time
+
+-   log(T) is the logarithm of the survival time
+
+-   X is a matrix of predictor variables
+
+-   β is a vector of coefficients representing the effects of the predictor variables on the logarithm of the survival time
+
+-   σ is a scaler quantity
+
+-   ε is the error term assumed to follow a specific distribution that depends on the assumed parametric form of the survival time.
+
+## Example of AFT model using "Log-Normal Distribution"
+
+```{r}
+library(survival)
+attach(lung)
+# Fit an AFT model using lognormal distribution
+model_aft <- survreg(Surv(time, status) ~ age + sex + ph.ecog, data = lung, dist = "lognormal")
+# Model summary
+summary(model_aft)
+```
+
+The summary output will provide the estimated coefficients, standard errors, and p-values for each predictor variable.
+
+## Acceleration Factor Calculation
+
+```{r}
+# Compute hazard ratios (exponentiated coefficients)
+hazard_ratios <- exp(coef(model_aft))
+hazard_ratios
+```
+
+## Interpretation
+
+-   For age, hazard ratio $< 1$ indicates that for each one-unit increase in age, the survival time is slowed down by a factor of 0.98 (or a 2% decrease).
+-   For sex, hazard ratio $> 1$ indicates that males have 68% higher hazard rate or accelerated survival time.
+-   A hazard ratio of ph.ecog $< 1$ suggests a 30% lower hazard rate or decelerated survival time associated with ph.ecog.
+
+## Plotting AFT Model Graphically
+
+```{r}
+suppressPackageStartupMessages({
+library(survival)
+library(survminer)
+library(ggplot2)
+})
+
+# Fit the AFT model on the lung dataset
+aft_model <- survreg(Surv(time, status) ~ age + sex + ph.ecog, data = lung, dist = "lognormal")
+
+# Create a new data frame with predicted survival times
+df <- data.frame(time = lung$time, age = lung$age, sex = lung$sex, ph.ecog = lung$ph.ecog, status=lung$status)
+df$surv_times <- predict(aft_model, newdata = df)
+
+# Plot the survival curves based on the AFT model
+ggsurvplot(survfit(Surv(surv_times, status) ~ 1, data = df),
+           data = df, xlab = "Time", ylab = "Survival Probability")
+```
+
+The survival curve plotted based on the AFT model for the lung dataset provides valuable insights into the estimated survival probabilities over time. The curve illustrates how the probability of survival changes as time progresses.
+
+## Example of AFT model using "Weibull Distribution"
+
+```{r}
+attach(ovarian)
+
+# Fit an AFT model using lognormal distribution
+model_aft_wb <- survreg(Surv(futime, fustat) ~ age + resid.ds + rx, data = ovarian, dist = "weibull")
+# Model summary
+summary(model_aft_wb)
+```
+
+## Acceleration Factor Calculation
+
+```{r}
+# Compute hazard ratios (exponentiated coefficients)
+hazard_ratio <- exp(coef(model_aft_wb))
+hazard_ratio
+```
+
+## Interpretation
+
+-   For age, a hazard ratio of 0.93 indicates that for each one-unit increase in age, the survival time is decelerated by a factor of 0.93(or a 7% decrease)
```

Comment:
For age, **an acceleration factor** <1 indicates that for each one-unit increase in age, the survival time is increased by a factor of 0.98 (or a 2% increase in survival time).